sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -544,6 +544,37 @@ SIREPO.app.directive('exportPythonLink', function(appState, panelState) {
     };
 });
 
+SIREPO.app.directive('randomSeed', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            field: '=',
+            form: '=',
+            max: '=',
+            model: '=',
+            modelName: '=',
+            viewName: '=',
+        },
+        template: `
+            <div class="col-sm-3">
+                <input data-string-to-number="integer" data-ng-model="model[field]" data-min="0" data-max="max" class="form-control" style="text-align: right" data-lpignore="true"/>
```

Comment:
2 spaces

Reply:
Also, when should we use lpignore? This is the first time I've noticed it and I see it is on other input tags.

Reply:
9abe9da

As for lpignore, all I know is that it blocks LastPass from auto-filling fields. I just copy-pasted it in; I don't think we have any guidelines as to when we actually need it